crates/testlang/outputs/npm/package/wasm/generated/interfaces/nomic-foundation-slang-cst.d.ts

```diff
@@ -20,18 +20,131 @@ export namespace NomicFoundationSlangCst {
  * These are nodes that can have child nodes and represent higher-level language constructs.
  */
 export declare enum NonterminalKind {
+  /**
+   * This kind represents a `AdditionExpression` node, with the following structure:
+   *
+   * ```ebnf
+   * (* Left-associative binary operator *)
```

Comment:
the indentation is broken here, but not in `cst.wit`. I wonder if this is a `jco` bug?

Reply:
Not `jco` itself, but it seems like `wit-parser` strips whitespace from doc comments.

Reply:
I see. Thanks for confirming.
I wonder if it is possible to replace the leading whitespace (" ") with a visually similar character that `wit-parser` won't strip in the `cst.wit.jinja2`? for example NBSP ("&#x00A0;")? would that still work for markdown syntax highlighting?

Reply:
Thanks for confirming.

In that case, I suggest submitting a PR or an issue at `wit-parser`: https://github.com/bytecodealliance/wasm-tools/tree/main/crates/wit-parser
It might be easier to fix there than trying to find a workaround on our side.
It is owned by the same org that maintains `jco`, and we might be able to get a quick turnaround there.

Reply:
Not blocking. Created #1188 to track this.